### DISCUSSION TITLE
第4节课笔记(9班)

### DISCUSSION THREAD

**Discussion by RangeKing** [InternLM/Tutorial#68] [笔记]
请在下方提交第 4 节课笔记笔记链接，知乎/CSDN/飞书 等等都可。
笔记让知识记得更牢固💪

**Comment by liuxianyi** [comment]
[Xtuner大模型微调工具](https://liuxianyi.notion.site/XTuner-800a548e6822495e848d7c7e3041c747?pvs=4)

**Comment by xiaomile** [comment]
[XTuner 大模型单卡低成本微调实战](https://blog.csdn.net/weixin_42567071/article/details/135542041)

**Comment by SaigeRing** [comment]
https://zhuanlan.zhihu.com/p/677700541

**Comment by ShelterWFF** [comment]
class4: http://t.csdnimg.cn/o4J7m

**Comment by li4692625** [comment]
https://blog.csdn.net/li4692625/article/details/135571786

**Comment by AIzealotwu** [comment]
https://blog.csdn.net/m0_72806612/article/details/135574724?spm=1001.2014.3001.5501

**Comment by xingyuan3** [comment]
笔记：https://www.mubu.com/doc/5qe1Pz3vzTd

**Comment by nxznm** [comment]
https://zhuanlan.zhihu.com/p/677742755

**Comment by KJ-Huang-sz** [comment]
https://blog.csdn.net/qq_40831032/article/details/135538834

**Comment by zhangzhidaSunny** [comment]
第4课笔记：
https://blog.csdn.net/m0_65719612/article/details/135582817

**Comment by wocclyl** [comment]
第四课笔记：
https://m1ti1x9upsq.feishu.cn/wiki/O3QTwoE7niXd9rkSTefczkzLnCh?from=from_copylink

**Comment by nini0919** [comment]
https://morning-gaura-eed.notion.site/04-a6aec88171024b6fa8baaeb31029c388?pvs=4

**Comment by haizeigh** [comment]
https://note.youdao.com/s/bqto3ZSz
xtuner微调 https://www.bilibili.com/video/BV1yK4y1B75J/?vd_source=d6b5a09dda93b2c223da08c9b8241a1f

**Comment by wangxuejiao** [comment]
[第四课笔记](https://zhuanlan.zhihu.com/p/677763102)
